python/lsst/ap/association/packageAlerts.py

```diff
@@ -368,8 +370,13 @@ def produceAlerts(self, alerts, visit, detector):
             Visit and detector ids of these alerts. Used to write out alerts
             which fail to be sent to the alert stream.
         """
+        # Assign a schema id using the schema name.
+        numbers = re.findall(r'\d+', self.alertSchema.definition['name'])
```

Comment:
As mentioned on the `alert_packet` review, let's use a function defined on `Schema`  here so you can just do `schema_id = self.alertSchema.get_schema_id()` or similar